app/src/main/java/com/dessalines/thumbkey/keyboards/DE_MessageEaseV1.kt

```diff
@@ -652,8 +654,14 @@ val MESSAGEEASE_DE_SHIFTED = KeyboardC(
     ),
 )
 
-val MESSAGEEASE_DE_KEYBOARD_MODES: Map<KeyboardMode, KeyboardC> = mapOf(
-    KeyboardMode.MAIN to MESSAGEEASE_DE_MAIN,
-    KeyboardMode.SHIFTED to MESSAGEEASE_DE_SHIFTED,
-    KeyboardMode.NUMERIC to NUMERIC_KEYBOARD,
-)
+val KB_DE_MESSAGEEASE_V1: KeyboardDefinition = KeyboardDefinition(
+    title = "DE MessageEase deutch v1",
+    modes = KeyboardDefinitionModes (
+        main = KB_DE_MESSAGEEASE_V1_MAIN,
+        shifted = KB_DE_MESSAGEEASE_V1_SHIFTED,
+        numeric = NUMERIC_KEYBOARD,
+    ),
+    settings = KeyboardDefinitionSettings(
+        autoCapitalizers = arrayOf()
```

Comment:
This should probably be an optional array, so definition files don't have to include this unless necessary.